modules/virt-deleting-vm-snapshot-cli.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-offline-vm-snapshots.adoc
+
+[id="virt-deleting-vm-snapshot-cli_{context}"]
+= Deleting a virtual machine snapshot in the CLI
+
+You can delete an existing virtual machine (VM) snapshot by deleting the appropriate `VirtualMachineSnapshot` object.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Delete the `VirtualMachineSnapshot` object. The snapshot controller deletes the `VirtualMachineSnapshot` along with the associated `VirtualMachineSnapshotContent` object.
++
+[source,terminal]
+----
+$ oc delete vmsnapshot <my-vmsnapshot>
+----
+
+.Verification steps
+
+. Verify that the snapshot is deleted and no longer attached to this VM:
```

Comment:
Same here - If the procedure is just a single step, this should be an unnumbered bullet.

Reply:
Fixed